CHANGES.md

```diff
@@ -1,3 +1,9 @@
+# 0.20.1
+
+-   Added a warning and error hook to the `Subform`, `RepearingForm`,
```

Comment:
RepearingForm

Reply:
You made more changes than this. You also made it impossible to set errors directly on repeating forms and such apart from the hook. Since that's a major change in behavior you should at least mention it.